src/components/navBar/index.tsx

```diff
@@ -1,17 +1,68 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
-
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
 const RDSLogo = '/RDSLogo.png';
+const edit = '/pencil.webp';
+
+function debounce<A = unknown, R = void>(
```

Comment:
Why is this `unknown`?

Reply:
made a custom hook and changed its type here https://github.com/Real-Dev-Squad/website-status/pull/278